index.html

```diff
@@ -1,69 +1,73 @@
+            <div>
+                <div class='home-btn'>
+                    <span class='ej-icon-Bento' style='font-size: 18px;'></span>
                 </div>
-                <div style="display: table-cell; width: 100%; text-align: center">
+                <div style='margin-left: 15px;display: inline-block'>
+                    <h1 class='home-btn-text'>Webmail</h1>
                 </div>
-                <div id="notification-div" class="header-right-pane">  
-                    <div style="pointer-events: none;">
-                        <button id='btnNotification' style="min-width: 52px"></button>
-                    </div>
+            </div>
+            <div style="display: table-cell; width: 100%; text-align: center">
+            </div>
+            <div id="notification-div" class="header-right-pane">
+                <div style="pointer-events: none;">
+                    <button id='btnNotification' style="min-width: 52px"></button>
                 </div>
-                <div id="settings-div" class="header-right-pane">  
-                    <div style="pointer-events: none;">
-                        <button id='btnSettings' style="min-width: 52px"></button>
-                    </div>
+            </div>
+            <div id="settings-div" class="header-right-pane">
+                <div style="pointer-events: none;">
+                    <button id='btnSettings' style="min-width: 52px"></button>
                 </div>
-                <div id="about-div" class="header-right-pane">  
-                    <div style="pointer-events: none;">
-                        <button id='btnAbout' style="min-width: 52px"></button>
-                    </div>
+            </div>
+            <div id="about-div" class="header-right-pane">
+                <div style="pointer-events: none;">
+                    <button id='btnAbout' style="min-width: 52px"></button>
                 </div>
-                <div id="profile-div" class="header-right-pane"> 
-                    <div style="right: 0px;position: absolute;top: 3px;pointer-events: none;margin-right: 5px;">
-                        <div class='logo logo-style1 profile-pic'></div>
-                    </div>  
-                    <div style="margin-right: 44px; pointer-events: none;">
-                        <button id='btnLoginName' class="btn-profile" style="min-width: 88px;font-weight: normal"></button>
-                    </div> 
+            </div>
+            <div id="profile-div" class="header-right-pane "> 
+                <div style=" pointer-events: none;">
+                    <button id="btnLoginName" class="btn-profile e-control e-btn e-lib btn-shadow-hide" style="min-width: 88px;font-weight: normal">Andrew Fuller</button>
                 </div>
+                <div style="pointer-events: none;margin-right: 5px;">
+                    <img class="e-avatar e-avatar-large e-avatar-circle" style="height: auto; width: auto;" src="./styles/images/1.png">
```

Comment:
Use SPAN tag with background image instead of IMG tag

Reply:
SPAN tag has been used.